modules/data.land/R/download_vswc.R

```diff
@@ -0,0 +1,59 @@
+#' Download Soil Moisture (VSWC)
+#' @title download_vswc
+#' @name download_vswc
+#' @param outdir location where you want the soil moisture ncdf file saved
+#' @param site_ID the NEON site ID
+#'
+#' @return a ncdf file with the soil moisture value for the most recent available day
+#' @author Christina Francis and Alexis Helgeson
+#'
+#' @examples outdir <- "/projectnb/dietzelab/cfranci1/soilmoist" site_ID <- "BART"
+
+download_vswc <- function(outdir, site_ID) {
```

Comment:
feel like the name should be more specific since this only get soil moisture from one source (NEON data as processed by EFI challenge)